src/wp-includes/class-wp-theme-json.php

```diff
@@ -75,6 +75,8 @@ class WP_Theme_JSON {
 	 * This contains the necessary metadata to process them:
 	 *
 	 * - path       => where to find the preset within the settings section
++	 * - override   => whether a theme preset with the same slug as a default preset
```

Comment:
There're a couple of extra `+` here. I've sent a patch to fix that and other minor changes at https://github.com/WordPress/wordpress-develop/pull/1967#issuecomment-981965618

Reply:
I've also prepared this Gutenberg PR https://github.com/WordPress/gutenberg/pull/36973 to make sure that the order of constants, methods, etc. as well as the PHPDoc comments are the same between the Gutenberg's `lib/class-wp-theme-json-gutenberg.php` and WordPress core `wp-includes/class-wp-theme-json.php`.

This way the backports are way easier and require minimal manual intervention.